pwa/app/components/tba/auth/signInButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { AuthProvider, signInWithPopup } from 'firebase/auth';
+
+import { auth } from '~/firebase/firebaseConfig';
+
+interface SignInButtonProps {
+  provider: AuthProvider;
+  logo: string;
+  buttonText: string;
+  buttonStyles: string;
+}
+
+const SignInButton = ({
```

Comment:
Generally we've been writing components as function objects rather than const lambdas, with props inline:

```ts
export default function SignInButton({provider, logo, buttonText, className}: {
  provider: AuthProvider;
  logo: string;
  buttonText: string;
  className: string;
}) {
  ....
}
```


```